wolox-training/mr-rails
# Mail articles by last 5 users

# Summary
* Added functionality to send email with last 10 articles from last 5 users

# Test plan
1. Start the sidekiq process with **bundle exec sidekiq**
2. Open the home page
3. Click on the "Send last articles from last 5 users by email" link
4. Check your email

# Trello card
https://trello.com/c/ZFZyEKuR/15-crear-un-email-con-los-ultimos-10-articulos-creados-por-las-5-ultimas-personas-en-registrarse
